### PR TITLE
Resolves #115

### LINKED ISSUE #115: Resolving cards sometimes doesn't clear 9 limitation
Not sure the exact steps to reproduce, but they were something along the lines of:

- Player A plays a 9, targeting an in-play Queen
- Player B plays a 6's one-off, removing some cards
- Player A plays a Jack, targeting an in-play 10
- Player B cannot replay their bounced Queen

### PR DESCRIPTION
**Pull request by byarmis**
Added test for reported issue.

It was pointed out to me after the tests were written, but these are from the point of view of the player who didn't play the 9

### CODE REVIEW

File: tests/e2e/specs/in_game/oneOffs.spec.js [hunk 1]
```diff
@@ -823,6 +822,112 @@ describe('Playing NINES', ()=>{
...
 			);
 		});
+
+		it('Plays a nine as a ONE-OFF, make sure that the bounced card is playable later', ()=>{
+			/*
+			1). P0 plays a 9, targeting an in-play Queen
+			2). P1 plays a 6's one-off, removing some cards
+			3). P0 plays a Jack, targeting an in-play 10
+			4). P1 couldn't replay their bounced Queen
+			 */
```
Comment (itsalaidbacklife) [inline tests/e2e/specs/in_game/oneOffs.spec.js:832]: With the bug fixed, rewrite step 4) to reflect the intended behavior, not the bug

File: tests/e2e/specs/in_game/oneOffs.spec.js [hunk 2]
```diff
@@ -823,6 +822,112 @@ describe('Playing NINES', ()=>{
...
+				p0FaceCards: [Card.KING_OF_HEARTS],
+				p1Hand: [Card.SIX_OF_HEARTS, Card.QUEEN_OF_HEARTS],
+				p1Points: [Card.TEN_OF_HEARTS],
+				p1FaceCards: [Card.QUEEN_OF_DIAMONDS],
+			});
+			cy.get('[data-player-hand-card]').should('have.length', 2);
+			cy.log('Loaded fixture');
+
+			// STEP 1
+			cy.log('STEP 1- P0 plays nine, targeting the Queen in play')
```
Comment (itsalaidbacklife) [inline tests/e2e/specs/in_game/oneOffs.spec.js:847]: ;

File: tests/e2e/specs/in_game/oneOffs.spec.js [hunk 3]
```diff
@@ -823,6 +822,112 @@ describe('Playing NINES', ()=>{
...
+					p0FaceCards: [Card.KING_OF_HEARTS],
+					p1Hand: [Card.SIX_OF_HEARTS, Card.QUEEN_OF_HEARTS, Card.QUEEN_OF_DIAMONDS],
+					p1Points: [Card.TEN_OF_HEARTS],
+					p1FaceCards: [],
+					scrap: [Card.NINE_OF_SPADES],
+				}
+			);
+
+			// STEP 2
+			cy.log('STEP 2- P1 plays a six, removing face cards')
```
Comment (itsalaidbacklife) [inline tests/e2e/specs/in_game/oneOffs.spec.js:873]: ;

File: tests/e2e/specs/in_game/oneOffs.spec.js [hunk 4]
```diff
@@ -823,6 +822,112 @@ describe('Playing NINES', ()=>{
...
+					p0FaceCards: [],
+					p1Hand: [Card.QUEEN_OF_HEARTS],
+					p1Points: [],
+					p1FaceCards: [Card.QUEEN_OF_DIAMONDS],
+					scrap: [Card.NINE_OF_SPADES, Card.KING_OF_HEARTS, Card.SIX_OF_HEARTS],
+				}
+			);
+
+
+		})
```
Comment (itsalaidbacklife) [inline tests/e2e/specs/in_game/oneOffs.spec.js:930]: ;

**Review by itsalaidbacklife — CHANGES_REQUESTED** [review]
Nice work; requesting a few chores

File: tests/e2e/specs/in_game/oneOffs.spec.js [hunk 5]
```diff
@@ -823,6 +822,223 @@ describe('Playing NINES', ()=>{
...
+					p1Points: [],
+					p1FaceCards: [Card.QUEEN_OF_DIAMONDS],
+					scrap: [Card.NINE_OF_SPADES, Card.KING_OF_HEARTS, Card.SIX_OF_HEARTS],
+				}
+			);
+
+
+		});
+
+		it.only('Plays a nine as a ONE-OFF, make sure that the bounced card is playable even if there is countering', ()=>{
```
Comment (itsalaidbacklife) [inline tests/e2e/specs/in_game/oneOffs.spec.js:932]: Remove .only()

**Review by itsalaidbacklife — CHANGES_REQUESTED** [review]
Looks good. Remove the .only() and we're good to merge

File: tests/e2e/specs/in_game/oneOffs.spec.js (hunk 5 above)
Comment (byarmis) [reply tests/e2e/specs/in_game/oneOffs.spec.js]: agh crap sorry, good catch

**Pull request merged**